packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -124,7 +124,10 @@ def get_create_batch_transaction(  # pylint: disable=unused-argument
             )
             tx = ledger_api.update_with_gas_estimate(tx)
             return tx
-        if ledger_api.identifier in [CosmosApi.identifier, FetchAIApi.identifier]:
+        if ledger_api.identifier in [
+            CosmosApi.identifier,
+            FetchAIApi.identifier,
+        ]:  # pragma: nocover # ledger not supported
```

Comment:
tests are disabled for the moment

according to https://github.com/valory-xyz/open-aea/blob/main/packages/fetchai/contracts/erc1155/tests/test_contract.py#L650